.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "meteor"]
+	path = meteor
+	url = git@github.com:meteor/meteor.git
```

Comment:
Please use HTTPS-based URLs for submodules always. Otherwise they fail if you do not have SSH keys setup.